test/test-network/systemd-networkd-tests.py

```diff
@@ -3481,6 +3491,60 @@ def test_ipv6_proxy_ndp(self):
         for i in range(1, 5):
             self.assertRegex(output, f'2607:5300:203:5215:{i}::1 *proxy')
 
+    def test_ipv6_retrans_time(self):
+        link='test25'
+        copy_network_unit('25-dummy.netdev', '25-dummy.link', copy_dropins=False)
+
+        start_networkd()
+        self.wait_operstate(link, 'off', setup_state='unmanaged')
+        # self.wait_online([f'{link}:degraded'])
+
+        output = check_output(f'ip link show dev {link}')
+        print(output)
+
+        # expect retrans_time_ms updated
+        copy_network_unit('25-ipv6-retrans-time-3s.network')
+        networkctl_reload()
+        self.check_ipv6_neigh_sysctl_attr(link, 'retrans_time_ms', '3000')
+        remove_network_unit('25-ipv6-retrans-time-3s.network')
+
+        # expect retrans_time_ms unchanged
+        copy_network_unit('25-ipv6-retrans-time-0s.network')
+        networkctl_reload()
+        self.check_ipv6_neigh_sysctl_attr(link, 'retrans_time_ms', '3000')
+        remove_network_unit('25-ipv6-retrans-time-0s.network')
+
+        # expect retrans_time_ms unchanged
+        copy_network_unit('25-ipv6-retrans-time-toobig.network')
+        networkctl_reload()
+        self.check_ipv6_neigh_sysctl_attr(link, 'retrans_time_ms', '3000')
+        remove_network_unit('25-ipv6-retrans-time-toobig.network')
+
+        # expect retrans_time_ms unchanged
+        copy_network_unit('25-ipv6-retrans-time-invalid.network')
+        networkctl_reload()
+        self.check_ipv6_neigh_sysctl_attr(link, 'retrans_time_ms', '3000')
+        remove_network_unit('25-ipv6-retrans-time-invalid.network')
+
+        # expect retrans_time_ms updated
+        copy_network_unit('25-ipv6-retrans-time-4s.network')
+        networkctl_reload()
+        self.check_ipv6_neigh_sysctl_attr(link, 'retrans_time_ms', '4000')
+        remove_network_unit('25-ipv6-retrans-time-4s.network')
+
+        # expect retrans_time_ms unchanged
+        networkctl_reload()
+        self.check_ipv6_neigh_sysctl_attr(link, 'retrans_time_ms', '4000')
+
+        # test is finished, final cleanup
+        remove_network_unit('25-dummy.netdev', '25-dummy.link')
+        networkctl_reload()
+
+        output = check_output(f'ip link show dev {link}')
+        print(output)
+
+        remove_link(link)
```

Comment:
These finalization step is not necessary. Please drop.

Reply:
Done.